### PR TITLE
SEA055 and SEA056 new mission yamls

### PR DESCRIPTION
**Pull request by akmazur**

### CODE REVIEW

File: mission_yaml/SEA55_M23.yml [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+  deployment_id: "23"
+  deployment_name: AUV_project
+  # date format 'yyyy-mm-dd'
+  deployment_start: "2021-08-21"
+  deployment_end: "2021-09-09"
+  project: AUV project
+  project_url: https://voiceoftheocean.org/samba-smart-autonomous-monitoring-of-the-baltic-sea/
+  summary: ' Part of AUV project'
+  # Examples: Skagerrak, Kattegatt, Baltic, Gulf of Bothnia, Gulf of Finland, Gulf of Riga
+  sea_name: Gotland
```
Comment (Maribaken) [inline mission_yaml/SEA55_M23.yml:46]: sea_name need to be something different. Probably "Baltic" as well. If we want to be specific we could change it to "Gotland Basin" or "Eastern Gotland Basin", but then we would nee to be specific in Bornholm as well, setting the sea_name to "Southern baltic proper" or similar.

Comment (Maribaken) [reply mission_yaml/SEA55_M23.yml:46]: I'd leave it as Baltic for now, if we decide to change it later we could trigger reprocessing then

File: mission_yaml/SEA55_M24.yml [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+  contributor_name: Callum Rollo, Louise Biddle, Olle Petersson, Aleksandra Mazur, Marcus Melin
+  contributor_role: Data Processor, PI, Head of Operations, Glider Technician, Glider Technician
+  creator_email: callum.rollo@voiceoftheocean.org
+  creator_name: Callum Rollo
+  creator_url: https://callumrollo.com/
+  deployment_id: "24"
+  deployment_name: SAMBA
+  # date format 'yyyy-mm-dd'
+  deployment_start: "2021-09-10"
+  deployment_end: "2021-10-05"
```
Comment (Maribaken) [inline mission_yaml/SEA55_M24.yml:41]: Minor issue, think this should be "2021-10-04"

File: mission_yaml/SEA55_M24.yml [hunk 3]
```diff
@@ -0,0 +1,456 @@
...
+  publisher_name:  Callum Rollo
+  publisher_url:   https://voiceoftheocean.org
+  references:     "created with pyglider https://github.com/c-proof/pyglider"
+  source:     Observational data from a profiling glider.
+  standard_name_vocabulary: CF STandard Name Table v49
+  transmission_system: IRIDIUM
+  glider_name: Kaprifol
+  glider_serial: "55"
+  wmo_id: "6801674"
+  comment: 'deployment and recovery in Skagerrak'
```
Comment (Maribaken) [inline mission_yaml/SEA55_M24.yml:31]: Deployment was in Bornholm according to glimpse

File: mission_yaml/SEA55_M24.yml [hunk 4]
```diff
@@ -0,0 +1,456 @@
...
+  deployment_id: "24"
+  deployment_name: SAMBA
+  # date format 'yyyy-mm-dd'
+  deployment_start: "2021-09-10"
+  deployment_end: "2021-10-05"
+  project: SAMBA
+  project_url: https://voiceoftheocean.org/samba-smart-autonomous-monitoring-of-the-baltic-sea/
+  summary: ' Part of SAMBA continuous monitoring'
+  # Examples: Skagerrak, Kattegatt, Baltic, Gulf of Bothnia, Gulf of Finland, Gulf of Riga
+  sea_name: Skagerrak
```
Comment (Maribaken) [inline mission_yaml/SEA55_M24.yml:46]: Deployment was in Baltic according to glimpse

**Review by Maribaken — CHANGES_REQUESTED** [review]
Some minor stuff

File: mission_yaml/SEA55_M23.yml (hunk 1 above)
Comment (akmazur) [reply mission_yaml/SEA55_M23.yml:46]: I agree. It should be Baltic

File: mission_yaml/SEA55_M24.yml (hunk 2 above)
Comment (akmazur) [reply mission_yaml/SEA55_M24.yml:41]: Corrected

File: mission_yaml/SEA55_M24.yml (hunk 3 above)
Comment (akmazur) [reply mission_yaml/SEA55_M24.yml:31]: Corrected

File: mission_yaml/SEA55_M24.yml (hunk 4 above)
Comment (akmazur) [reply mission_yaml/SEA55_M24.yml:46]: Corrected

**Review by Maribaken — APPROVED** [review]

**Review by Maribaken — APPROVED** [review]
All looks good, will merge this

**Pull request merged**